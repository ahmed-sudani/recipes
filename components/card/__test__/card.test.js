import { render, screen } from '@testing-library/react'
import Card from '..'

const card = {
  name: 'chechen',
  time: '50:00',
  img: '/cster.png',
  country: 'italy',
}
describe('Card', () => {
  it('should render card component', () => {
    render(<Card {...card} />)
    const cardElement = screen.getByTestId('card')
    expect(cardElement).toMatchSnapshot()
  })
})
