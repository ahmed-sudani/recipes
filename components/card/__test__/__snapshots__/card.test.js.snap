// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render card component 1`] = `
<div
  class="card"
  data-testid="card"
>
  <span
    style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
  >
    <span
      style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
    >
      <img
        alt=""
        aria-hidden="true"
        src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27300%27%20height=%27150%27/%3e"
        style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
      />
    </span>
    <img
      alt="chechen"
      data-nimg="intrinsic"
      decoding="async"
      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
    />
    <noscript />
  </span>
  <span
    class="time"
  >
    <svg
      fill="currentColor"
      height="1em"
      viewBox="0 0 16 16"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
      />
      <path
        d="M8 16A8 8 0 1 0 8 0a8 8 0 0 0 0 16zm7-8A7 7 0 1 1 1 8a7 7 0 0 1 14 0z"
      />
    </svg>
     
    50:00
  </span>
  <div
    class="cardBody"
  >
    <div
      class="cardHead"
    >
      <h3
        class="name"
      >
         
        chechen
      </h3>
      <div
        class="country"
      >
         
        italy
         
      </div>
      <svg
        class="heart"
        fill="currentColor"
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M8 1.314C12.438-3.248 23.534 4.735 8 15-7.534 4.736 3.562-3.248 8 1.314z"
          fill-rule="evenodd"
        />
      </svg>
    </div>
    <div
      class="description"
    >
      <div>
         
      </div>
      <div>
         
      </div>
      <div>
         
      </div>
    </div>
  </div>
</div>
`;
